frontend/src/components/dashboard/customHomePage/MineSubscriptionTable.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from "react";
+import PropTypes, { objectOf, string } from "prop-types";
+import { Link } from "react-router-dom";
+import { Table, Popconfirm } from "antd";
+import { uniqBy } from "lodash";
+import * as router from "@/constants/routes";
+import * as Strings from "@/constants/strings";
+import NullScreen from "@/components/common/NullScreen";
+import CustomPropTypes from "@/customPropTypes";
+import { UNSUBSCRIBE } from "@/constants/assets";
+
+/**
+ * @class MineSubscriptionTable is a user specific table of mines they have subscribed to with the ability to unsubscribe
+ *
+ */
+
+const propTypes = {
+  handleUnSubscribe: PropTypes.func.isRequired,
+  subscribedMines: PropTypes.arrayOf(CustomPropTypes.mine).isRequired,
+  mineRegionHash: objectOf(string).isRequired,
```

Comment:
If we're going to start importing specific named PropTypes functions to avoid prefixing  PropTypes.objectOf, can we please make it consistent and import func, arrayOf, etc as well? I'd prefer we keep the PropTypes prefix given how widely we have that in our code-base, or refactor it everywhere as a seperate PR.

Reply:
Agreed, overlooked that. I also prefer to keep the PropTypes prefix. Will update!